docs/05_community-developer-tools/index.md

```diff
@@ -76,7 +78,9 @@ If you would like to submit a tool to be added to the list, please fill in [this
 
 * [Scatter](https://github.com/GetScatter/ScatterDesktop): Scatter desktop is a cutting-edge wallet as well as a global local-machine authentication provider.
 
-* [Portis](https://docs.portis.io/#/): Portis is a "Non-Custodial Blockchain Wallet that Makes Apps Simple for Everyone"
+* [Portis](https://docs.portis.io/#/): Portis is a "Non-Custodial Blockchain Wallet that Makes Apps Simple for Everyone".
+
+* [Wombat](https://github.com/wombat-tech): Wombat is "the most user-friendly way of handling blockchain on Desktop, iOS & Android." and lets users "send, receive, and earn various tokens without any technical knowledge." 
```

Comment:
How about this:

_A user-friendly, secure wallet for EOS and Telos blockchains available on iOS, Android, and Desktop. You can send, receive, and earn various tokens without any technical knowledge._